examples/step_by_step_nav/frontend/learn_to_drive_a_car.json

```diff
@@ -11,6 +11,30 @@
   "base_path": "/learn-to-drive-a-car",
   "description": "Learn to drive a car in the UK - get a provisional licence, take driving lessons, prepare for your theory test, book your practical test.",
   "links": {
+    "pages_secondary_to_step_nav": [
+      {
+        "api_path": "/api/content/government/publications/car-and-small-van-driving-syllabus",
+        "base_path": "/government/publications/car-and-small-van-driving-syllabus",
+        "content_id": "5e5dc201-7631-11e4-a3cb-005056011aef",
+        "document_type": "guidance",
+        "locale": "en",
+        "public_updated_at": "2014-11-24T15:00:02.000+00:00",
+        "schema_name": "publication",
+        "title": "Learning to drive a car syllabus",
+        "withdrawn": false
```

Comment:
Nitpick - I think the only thing that's missing from this example is an empty links hash here, similar to `pages_part_of_step_nav` in https://www.gov.uk/api/content/learn-to-drive-a-car